### PR TITLE
Add user-agent with contact

### PR DESCRIPTION
**Pull request by jobergum**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by hmusum — DISMISSED** [review]
👍

File: config-proxy/src/main/java/com/yahoo/vespa/config/proxy/filedistribution/UrlDownloader.java [hunk 1]
```diff
@@ -22,11 +22,13 @@ class UrlDownloader implements Downloader {
 
     private static final Logger log = Logger.getLogger(UrlDownloader.class.getName());
     private static final String CONTENTS_FILE_NAME = "contents";
+    private static final String USER_AGENT_MODEL_DOWNLOADER = "Vespa.ai config-proxy url model downloader (https://github.com/vespa-engine/vespa)";
```
Comment (oyving) [inline config-proxy/src/main/java/com/yahoo/vespa/config/proxy/filedistribution/UrlDownloader.java:25]: Let's not block a fix behind this comment, but we should have a RFC compliant User-Agent string. E.g. `User-Agent: Vespa/<version> (model download - https://github.com/vespa-engine/vespa)`

Comment (jobergum) [reply config-proxy/src/main/java/com/yahoo/vespa/config/proxy/filedistribution/UrlDownloader.java]: Thanks, check now

**Review by bjorncs — APPROVED** [review]
Consider using `Vespa.VERSION` instead of hardcoded '8.x'

**Pull request merged**